src/display/annotation_layer.js

```diff
@@ -262,7 +262,9 @@ class AnnotationElement {
         quadPoint[1].y,
       ];
       this.data.rect = rect;
-      quadrilaterals.push(this._createContainer(ignoreBorder));
+      const quad = this._createContainer(ignoreBorder);
+      quad.className = "highlightAnnotation";
```

Comment:
The `_createQuadrilaterals` method is generic since multiple annotation types can use quadrilaterals, so this class name should not be set here. Fortunately, it can simply be fixed locally in the `HighlightAnnotationElement` class below.

Hence I'd suggest to revert all changes in this method and instead simply change the `render` method in the `HighlightAnntationElement` class to the following (untested):

```javascript
  render() {
    if (!this.data.hasPopup) {
      this._createPopup(null, this.data);
    }

    if (this.quadrilaterals) {
      this.quadrilaterals.forEach(quadrilateral => {
        quadrilateral.className = "highlightAnnotation";
      });
      return this.quadrilaterals;
    }

    this.container.className = "highlightAnnotation";
    return this.container;
  }
```

Reply:
@calixteman Hm, it looks like this comment was forgotten to be addressed. Would you mind following this up in a new PR to move this specific class name setting out of the generic function?

Reply:
Sorry misread the comment, thought it had been addressed. Filed #12585